plugin.go

```diff
@@ -101,8 +101,8 @@ func (c *Plugin) Serve() chan error {
 		Addr:                         c.cfg.Address,
 		Handler:                      mux,
 		DisableGeneralOptionsHandler: false,
-		ReadTimeout:                  time.Minute,
-		ReadHeaderTimeout:            time.Minute,
+		ReadTimeout:                  time.Duration(c.cfg.HealthCheckTimeout) * time.Second,
+		ReadHeaderTimeout:            time.Duration(c.cfg.HealthCheckTimeout) * time.Second,
 		WriteTimeout:                 time.Minute,
 		IdleTimeout:                  time.Minute,
```

Comment:
Maybe all 4 should have this value, but since we don't write anything, I'm guessing it doesn't matter?

Reply:
Yeah, does not matter.